plugin/src/main/java/org/wildfly/plugin/server/StartBootableJarMojo.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.wildfly.plugin.server;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugin.logging.Log;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+import org.wildfly.core.launcher.BootableJarCommandBuilder;
+import org.wildfly.core.launcher.CommandBuilder;
+import org.wildfly.plugin.common.Environment;
+import org.wildfly.plugin.common.PropertyNames;
+import org.wildfly.plugin.common.StandardOutput;
+import org.wildfly.plugin.common.Utils;
+import org.wildfly.plugin.provision.PackageServerMojo;
+import org.wildfly.plugin.tools.bootablejar.BootableJarSupport;
+
+/**
+ * Starts a WildFly Application Server packaged as Bootable JAR.
+ * <p/>
+ * The purpose of this goal is to start a WildFly Application Server packaged as a Bootable JAR for testing during the maven
+ * lifecycle.
+ */
+@Mojo(name = "start-bootable", requiresDependencyResolution = ResolutionScope.RUNTIME)
```

Comment:
I'm just thinking a bit out loud here. Should the goal name be `start-jar`? Just thinking of something short to type is all, but I'm not sure I like `start-jar` :)

Reply:
I much prefer start-jar, start-bootable is also a bit ambiguous, all server boot... So changed to start-jar.